aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js

```diff
@@ -70,7 +70,7 @@ const secondChildGrid = {
   gap: 'medium',
 };
 
-const PageContent = () => {
+export const PageContent = () => {
```

Comment:
Why are we exporting this?

Reply:
Im importing this for the first example of the page.

Reply:
Can we use the new grommet component instead?

Reply:
good point let me switch that out